<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IPConfigCmdEditor</class>
 <widget class="QWidget" name="IPConfigCmdEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>必填参数</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>无</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>可选参数</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0">
        <property name="horizontalSpacing">
         <number>20</number>
        </property>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox_showclassid">
            <property name="minimumSize">
             <size>
              <width>260</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>显示指定适配器的DHCP类别ID：</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_showclassid">
            <property name="toolTip">
             <string>空表示更新所有</string>
            </property>
            <property name="placeholderText">
             <string>适配器名称，支持通配符</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox_renew">
            <property name="minimumSize">
             <size>
              <width>260</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>更新指定适配器：</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_renew">
            <property name="toolTip">
             <string>空表示更新所有</string>
            </property>
            <property name="placeholderText">
             <string>适配器名称，支持通配符</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox_setclassid">
            <property name="minimumSize">
             <size>
              <width>260</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>设置指定适配器的DHCP类别ID：</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_setclassid">
            <property name="toolTip">
             <string>空表示更新所有</string>
            </property>
            <property name="placeholderText">
             <string>适配器名称，支持通配符</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_newClassid">
            <property name="placeholderText">
             <string>新的classid</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox_release">
            <property name="minimumSize">
             <size>
              <width>260</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>释放所有匹配的连接：</string>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_release">
            <property name="toolTip">
             <string>空表示更新所有</string>
            </property>
            <property name="placeholderText">
             <string>适配器名称，支持通配符</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_flushdns">
          <property name="text">
           <string>清理并重设DNS客户解析器缓存的内容</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_displaydns">
          <property name="text">
           <string>显示DNS客户解析器缓存的内容</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_all">
          <property name="text">
           <string>显示所有适配器的完整 TCP/IP 配置信息</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
